### PR TITLE
Create PO in cli

### PR DESCRIPTION
**Pull request by KevinAnsleyJohnson**
Adding functionality to create purchase order. Based on the updated RFC,
we need buyer-org and seller-org instead of owning org. Changes are
reflected in the man pages as well as code.

Signed-off-by: Kevin Johnson <kevin_johnson@cargill.com>

### CODE REVIEW

File: cli/Cargo.toml [hunk 1]
```diff
@@ -47,6 +47,7 @@ serde = { version = "1", features = ["derive"] }
 serde_yaml = "0.8"
 diesel_migrations = "1.4"
 whoami = "1"
+rand = "0.7"
```
Comment (dnewh) [inline cli/Cargo.toml:50]: This should be pulled in by the purchase-order feature

**Review by dnewh — CHANGES_REQUESTED** [review]

File: cli/src/main.rs [hunk 2]
```diff
@@ -1431,9 +1433,19 @@ fn run() -> Result<(), CliError> {
...
-                            Arg::with_name("org")
-                                .value_name("org_id")
-                                .long("org")
+                            Arg::with_name("buyer_org_id")
+                                .value_name("buyer_org_id")
+                                .long("buyer-org")
+                                .short("buyer")
+                                .takes_value(true)
+                                .required(true)
+                                .help("ID of the organization which owns the Purchase Order"),
```
Comment (shannynalayna) [inline cli/src/main.rs:1442]: Perhaps we could change this "help" text to be more specific for the buyer and seller org options

**Review by leebradley — APPROVED** [review]

File: cli/Cargo.toml (hunk 1 above)
Comment (KevinAnsleyJohnson) [reply cli/Cargo.toml]: fixed!

File: cli/src/main.rs (hunk 2 above)
Comment (KevinAnsleyJohnson) [reply cli/src/main.rs]: fixed!

**Review by shannynalayna — APPROVED** [review]

**Review by dnewh — APPROVED** [review]

**Pull request merged**